Dockerfile

```diff
@@ -1,16 +1,17 @@
-FROM golang:1.20-alpine3.18 as builder
+FROM --platform=$BUILDPLATFORM golang:1.20-alpine3.18 as builder
 WORKDIR $GOPATH/src/go.k6.io/k6
 COPY . .
+ARG TARGETOS TARGETARCH
 RUN apk --no-cache add git=~2
-RUN CGO_ENABLED=0 go install -a -trimpath -ldflags "-s -w -X go.k6.io/k6/lib/consts.VersionDetails=$(date -u +"%FT%T%z")/$(git describe --tags --always --long --dirty)"
+RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -trimpath -o /usr/bin/k6
```

Comment:
nit: Is there a reason we want the `-a`?  I think it rebuilds the stdlib which seems .. pointless :shrug:

Reply:
I was commenting the same, but you were faster!!

I researched about it and I didn't find any reason, I guess there was some historical reason for it considering how old this command is.
I'm going to remove it as it isn't needed at the moment.